cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/StreamSizeScheduler.java

```diff
@@ -1001,6 +1034,8 @@ public void receivedPollingInformation(@Nonnull StreamSize pollingInfo) {
       argsBuilder.put(ProgramOptionConstants.RUN_DATA_SIZE, Long.toString(pollingInfo.getSize()));
```

Comment:
So add everything to system overrides in both `StreamSizeScheduler` and `TimeScheduler`?